src/tests/ut/calendar/calendar_test.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "calendar.hpp"
+#include "clinic_facade.hpp"
+#include "day.hpp"
+#include "day_name.hpp"
+#include "month.hpp"
+#include "year.hpp"
+#include "gtest/gtest.h"
+#include <chrono>
+
+class CalendarTestFixture : public ::testing::Test
+{
+  public:
+    std::shared_ptr<Calendar> cal;
```

Comment:
1. name "cal" doesn't say much.
A popular convention of sut_ (as for System Under Test) might be used.
Or calendar_tested. Something meaningful.
2. please add _ at the end in order to stick class member naming convention